### ISSUE TITLE
jQuery not loading

### ISSUE THREAD

**Issue by ursus-the-bear** [zeromancer1972/FileSilo#13]
After trying to install for a couple of days now I still do not have a working system. To get the site working I have to hardcode all the links in the theme. I then get the error that the application cannot find jQuery. 

<img width="844" alt="Screenshot 2019-05-11 at 18 15 18" src="https://user-images.githubusercontent.com/5219580/57572325-d8fce400-7418-11e9-9c9c-16d231a2daff.png">

If I do not correct the URLs in the theme I get:

<img width="807" alt="Screenshot 2019-05-11 at 18 17 40" src="https://user-images.githubusercontent.com/5219580/57572355-3bee7b00-7419-11e9-93c5-61f0e69db361.png">

I have reinstalled the extlib, recreated the web site in names.nsf, created the .NTF from git and downloading from openNTF - does anybody have any other ideas?

**Comment by zeromancer1972** [comment]
Please deactivate resource aggregation in XSP properties to check which file is missing. Domino 10 reacts with a 404 when a aggregated resource is not found. 9.0.1 just loaded everything and skipped the missing file(s).

**Comment by ursus-the-bear** [comment]
weird -> 503 service unavailable? if I click on the link the file is opened.

<img width="1195" alt="Screenshot 2019-05-11 at 21 15 07" src="https://user-images.githubusercontent.com/5219580/57574045-043ffd00-7432-11e9-83d7-a283ccc04786.png">

**Comment by zeromancer1972** [comment]
Are you still accessing it via a proxy? Please check what happens if you connect directly to the Domino machine. Something's really weird on your system.

**Comment by ursus-the-bear** [comment]
Nope, same problem when I access it locally on the server -> I have tried it on two different servers. The other sites I have programmed myself (using Bootstrap from extlib) and FileSendr (https://www.openntf.org/p/filesendr by Declan) are running fine. I wanted to replace FileSendr with a project that is being actively maintained 😊

<img width="1293" alt="Screenshot 2019-05-12 at 11 12 11" src="https://user-images.githubusercontent.com/5219580/57580272-d1d0e700-74a7-11e9-83f1-f49150a1dd63.png">
